packages/hash/api/jest.config.js

```diff
@@ -1,5 +1,8 @@
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
 module.exports = {
+  moduleNameMapper: {
+    "@hashintel/hash-shared(.*)": "<rootDir>/../shared/src$1",
+  },
```

Comment:
What's this for?

Reply:
This is to allow the jest tests to import utilities from the shared project through workspace references. 
With the decision to let the frontend handle restructuring the tree from references, I moved `treeFromParentReferences` to the shared utilities file, the same function is re-exported in the API util file, but the tests are still at the API location

Reply:
Understood, thank you